src/image.jl

```diff
@@ -43,19 +39,14 @@ end
 Get image dimensions (or `i`th dimension), without reading the image
 into memory.
 """
-function size(hdu::ImageHDU)
+function size(hdu::ImageHDU{<:Any,N}) where N
     fits_assert_open(hdu.fitsfile)
     fits_movabs_hdu(hdu.fitsfile, hdu.ext)
     sz = fits_get_img_size(hdu.fitsfile)
-    tuple(sz...)
+    ntuple(i -> sz[i], N)
```

Comment:
@jishnub I assume this was changed due to a type-instability/speedup, but it obfuscates the code. Does `Tuple(sz)` match performance? If so I would prefer that for clarity

Reply:
This may be simplified to `NTuple{N,eltype(sz)}(sz)` or to `NTuple{N,Int}(sz)` with full type inference. `Tuple(sz)` will not lead to type stability as the number of elements is not known.

Reply:
I've changed this to `NTuple{N,Int}` in 207fd27. I chose the return type of `Int` instead of `Cint` to parallel `size` for `AbstractArray`s